packages/table/src/table.scss

```diff
@@ -20,13 +20,23 @@ $menu-z-index: $row-z-index + 1 !default;
 .bp-table-container {
   display: flex;
   flex-direction: column;
+  // translateZ(0) has no visual effect, but it achieves the following:
+  //   - creates a new stacking context
+  //   - moves this element to a new layer to enable hardware-accelerated animation
+  //
+  // `will-change: transform` does the same thing, but it suffers from a Chrome bug that
+  // makes the Table blurry when transform'd (e.g. when the Table is in a Popover that
+  // animates open). we should move to `will-change` once that bug is fixed.
+  //
+  // further reading: https://dev.opera.com/articles/css-will-change-property/
+  transform: translateZ(0);
```

Comment:
Happy to add a mixin. Note that this `transform` rule doesn't just create a new stacking context; it also creates a new "containing block" (meaning that fixed-position elements within the block will be transformed as well). I think it'd be clearest to name the mixin `force-hardware-acceleration()`, because that's what it's really meant for.